packages/suite/src/actions/wallet/transactionActions.ts

```diff
@@ -174,3 +182,32 @@ export const fetchTransactions = (account: Account, page: number, perPage?: numb
         });
     }
 };
+
+// Fetch all pages
+export const fetchAllTransactions = (account: Account) => async (dispatch: Dispatch) => {
+    const perPage = 500;
+    const totalPages = Math.ceil(account.history.total / perPage);
```

Comment:
this will not work for XRP (no paging there)

Reply:
Addressed in 06c849f.